core/agents/troubleshooter.py

```diff
@@ -33,7 +33,29 @@ class Troubleshooter(IterationPromptMixin, BaseAgent):
     agent_type = "troubleshooter"
     display_name = "Troubleshooter"
 
-    async def run(self) -> AgentResponse:
+    async def run(self):
```

Comment:
`run()` should return AgentResponse, when is that not the case? From what I see, we do return AgentResponse always but should double check

Reply:
you're right, fixing